cmake/packaging/rpm/Changelog.txt

```diff
@@ -0,0 +1,2 @@
+* Wed Oct 24 2018 STE||AR Group <contact@stellar-group.org> 1.2.0-1
+- HPX Release 1.2
```

Comment:
Does this need to be updated for every release? If so, could you add it to the release procedure?

Reply:
Could we add a link to some (permanent) webpage that refers to the latest release notes?

Reply:
@hkaiser Is this sufficient?
```

* Wed Oct 24 2018 STE||AR Group <contact@stellar-group.org> 1.2.0-1
- HPX Release 1.2 (https://stellar-group.github.io/hpx/docs/sphinx/tags/1.2.0/html/releases/whats_new_1_2_0.html)
```

Reply:
@msimberg Would be great to add it to the release procedure and update the version number and the link to the release notes.